src/actions/actionProperties.tsx

```diff
@@ -290,6 +290,19 @@ export const actionChangeFontSize = register({
       }),
       appState: {
         ...appState,
+        editingElement: appState.editingElement
+          ? {
+              ...(appState.editingElement as ExcalidrawTextElement),
+              isDeleted: false,
+              font: `${value}px ${
+                appState
+                  ? (
+                      appState.editingElement as ExcalidrawTextElement
+                    ).font.split("px ")[1]
+                  : ""
+              }`,
+            }
+          : null,
```

Comment:
drop this. `editingElement` is a reference to the text element that's updated from this action already

Reply:
So the flow is: 

- when change font size and there is an active editing element, we change the properties of the editing element
- these changes on the editing element, will find their way inside the contenteditable div so they can be applied while is still editable
- when we click anywhere outside the editable div, we update any changes that happened inside to the outside world so they can be applied to the actual elements

Reply:
Forgot we're creating a new element on prop change. That's a problem. But instead of mutating the `appState.editingElement`, we need to renew the reference. I'm figuring out how best to do this.